index.src.html

```diff
@@ -352,32 +344,18 @@ <h3 id="header">
   [[!RFC5234]]:
 
   <pre class="abnf" link-type="grammar" dfn-type="grammar">
-    Report-To = <a>json-field-value</a>
-                ; See Section 2 of [[HTTP-JFV]], and Section 2 of [[RFC7159]]
+    Clear-Site-Data = "Clear-Site-Data" ":" <a>clear-site-data-value</a>
+    <dfn>clear-site-data-value</dfn> = OWS 1#(<a>type</a> OWS)
+    <dfn>type</dfn> = atom
```

Comment:
Ah, alright. I must have read something older then. Updated.

I'm now linking to 7230 for `#rule`, since it's more detailed (e.g. contains the discussion about OWS). That is helpful at least for a reader like myself.